staging/kos/cmd/controller-manager/main.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"math/rand"
+	"os"
+	"os/signal"
+	"runtime"
+	"syscall"
+	"time"
+
+	"github.com/golang/glog"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"k8s.io/examples/staging/kos/cmd/controller-manager/options"
+	kosclientset "k8s.io/examples/staging/kos/pkg/client/clientset/versioned"
+	kosinformers "k8s.io/examples/staging/kos/pkg/client/informers/externalversions"
+)
+
+func main() {
+	controllersOptions := &options.KOSControllerManagerOptions{
+		SubnetValidationController: &options.SubnetValidationControllerOptions{},
+		IPAMController:             &options.IPAMControllerOptions{},
+	}
+	controllersOptions.AddFlags()
+	flag.Set("logtostderr", "true")
+	flag.Parse()
+
+	if len(os.Getenv("GOMAXPROCS")) == 0 {
+		runtime.GOMAXPROCS(runtime.NumCPU())
+	}
+	rand.Seed(time.Now().UnixNano())
+	rand.Uint64()
+	rand.Uint64()
+	rand.Uint64()
+
+	baseClientCfg, err := clientcmd.BuildConfigFromFlags("", controllersOptions.KubeconfigFilename)
+	if err != nil {
+		glog.Errorf("Failed to build client config for kubeconfig=%q: %s", controllersOptions.KubeconfigFilename, err.Error())
+		os.Exit(2)
+	}
+
+	kosInformersClientCfg := *baseClientCfg
```

Comment:
Before this PR, each controller sets its client QPS and burst. That client is then used both directly by the controller and by its Informers. After this PR, all controllers get separate clients with separate QPS and bursts, but they share the same Informer. Not sure whether we want CLI flags to set QPS and burst for the client used by the Infromers.